modules/microshift-about.adoc

```diff
@@ -7,9 +7,10 @@
 
 Working with resource-constrained field environments and hardware presents many challenges not experienced with cloud computing. {product-title} enables you to solve problems for edge devices by:
 
+* Running the same Kubernetes workloads you run in the cloud, but at the edge.
 * Overcoming the operational challenge of minimal system resources, for example, a {op-system-chip}.
 * Addressing the environmental challenges of severe networking constraints such as low or no connectivity.
 * Meeting the physical constraint of hard-to-access locations by installing your system images directly on edge devices.
-* Building on and integrating with edge-optimized operating systems such as {op-system-first}.
+* Building on and integrating with edge-optimized operating systems such as {op-system-ostree-first}.
 
-{product-title} has the simplicity of single-node deployment with the functions and services you need for computing in resource-constrained locations. You can have many deployments on different hosts, creating the specific system image needed for each of your applications. Installing {product-title} on top of your managed {op-system} devices in hard-to-service locations also allows for streamlined over-the-air updates.
+{product-title} has the simplicity of single-node deployment with the functions and services you need for computing in resource-constrained locations. You can have many deployments on different hosts, creating the specific system image needed for each of your applications. Installing {product-title} on top of your managed {op-system} devices in hard-to-service locations also allows for over-the-air updates.
```

Comment:
I'm not sure the over-the-air nature of the updates is the most important to call out. Standard RHEL can be updated that way, too. The thing ostree gives is atomic updates and rollbacks. Maybe there's some wording about that in the RHEL docs that we can reuse?

Reply:
Are those features available at Dev Preview and Tech Preview? I thought they were not since we have warnings that new installs are needed pre-GA.

Reply:
Upgrade and rollback are features of the OS. We haven't done the integration to try to make them safe with MicroShift, which is why we say you have to reinstall.

How about just cutting the sentence for now?